args.js

```diff
@@ -0,0 +1,5 @@
+// the default argument setter, you can export your own argument-setter
+// by putting your array-returning function in your own args.js in your project's CWD
+module.exports = function() {
```

Comment:
can this live in args?  maybe do a cd test before npm test so we can put all of that in test directory